RELEASES.md

```diff
@@ -93,8 +93,10 @@ This may also contain some minor features, but ensure that it does NOT contain a
 - Create a tagged release from the release branch `release-vX.Y.x`. The release should start with "v" and be followed by the version number.
 
 ## RC Release
-1. Move on to [Releases > Draft a New Release](https://github.com/pipe-cd/pipecd/releases/new).
-2. Set values as below:
+
+1. (Only for the patch rc release) At first, we need to cherry-pick the fixes to the release branch. Please refer to [Backport fixes and Release note](https://github.com/pipe-cd/pipecd/blob/master/RELEASES.md#backport-fixes-and-release-note)
```

Comment:
```suggestion
1. Prepare: Ensure all changes you want to attach to the release are available in the release target branch (`master` for the minor, `release-vX.Y.x` for the patch). Please refer to [Backport fixes and Release note](#backport-fixes-and-release-note)
```
how about this?

Reply:
@khanhtc1202 
Thanks, It looks totally nice.  
`Backport fixes and Release note` is about cherry-pick process and need only for the patch release.
So how about this? Just added a little change.

```
Prepare: Ensure all changes you want to attach to the release are available in the release target branch (`master` for the minor, `release-vX.Y.x` for the patch). For the patch, please refer to [Backport fixes and Release note](#backport-fixes-and-release-note) 
```

Reply:
@ffjlabo thanks, it looks better 👍

Reply:
fixed! [4d2df7e](https://github.com/pipe-cd/pipecd/pull/5180/commits/4d2df7e039ee2d2e0d55704bc27441f547ae624b)